sql/catalyst/src/main/java/org/apache/spark/sql/connector/expressions/GeneralSQLExpression.java

```diff
@@ -22,20 +22,34 @@
 import org.apache.spark.annotation.Evolving;
 
 /**
- * The general SQL string corresponding to expression.
+ * The general V2 expression corresponding to V1 expression.
  *
  * @since 3.3.0
  */
 @Evolving
 public class GeneralSQLExpression implements Expression, Serializable {
```

Comment:
Can we rename it something like `SQLStringHolder`, `SQLStringRepresentation` and `SQLString`?

Reply:
We will use V2 expression to replace SQL string.

Reply:
Do you meant that we will use this for all V2 expressions? I thought this is one of the optional APIs.

Reply:
Currently, we use this for V2 expressions appears in aggregate.